src/Temporalio.Extensions.Hosting/TemporalWorkerServiceOptionsBuilderExtensions.cs

```diff
@@ -42,6 +42,39 @@ public static ITemporalWorkerServiceOptionsBuilder AddSingletonActivities(
             return builder.ApplyTemporalActivities(type);
         }
 
+        /// <summary>
+        /// Register the given type as a singleton if not already done and all operations of the
+        /// given type to the worker. Basically
+        /// <see cref="ServiceCollectionDescriptorExtensions.TryAddSingleton{TService}(IServiceCollection)" />
+        /// +
+        /// <see cref="ApplyNexusOperations{T}(ITemporalWorkerServiceOptionsBuilder)" />.
+        /// </summary>
+        /// <typeparam name="T">Service handler type for which operations are registered.</typeparam>
+        /// <param name="builder">Builder to use.</param>
+        /// <returns>Same builder instance.</returns>
+        /// <remarks>WARNING: Nexus support is experimental.</remarks>
+        public static ITemporalWorkerServiceOptionsBuilder AddSingletonOperations<T>(
```

Comment:
```suggestion
        public static ITemporalWorkerServiceOptionsBuilder AddSingletonNexusService<T>(
```

I think calling these methods "Nexus service" is clearer (though services are just collections of operations)

Reply:
Done